docs/resources/settings.md

```diff
@@ -0,0 +1,32 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "xray_settings Resource - terraform-provider-xray"
+subcategory: ""
+description: |-
+  Provides an Xray DB Sync Time resource.
+---
+
+# xray_settings (Resource)
+
+Provides an Xray DB Sync Time resource.
+
+## Example Usage
+
+```terraform
+resource "xray_settings" "db_sync" {
+  db_sync_updates_time = "18:40"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **db_sync_updates_time** (String) The time of the Xray DB sync daily update job.
+
+### Optional
+
+- **id** (String) The ID of this resource.
```

Comment:
mmm, no. Not optional - also not managable by them

Reply:
for some reason, that field was generated by `tfgeneratedocs` plugin. I'll remove it